README.md

```diff
@@ -100,6 +100,37 @@ To fix this, run the script:
 yarn build:local-packages
 ```
 
+#### WINDOWS Installation Notes
+
+ scoop tool installation:	// involves some powershell env setups
+  https://github.com/lukesampson/scoop/wiki/Quick-Start
```

Comment:
The problem you faced with setting up in Windows could also happen on other operating systems. What do you think about restructuring this to something like:

```
#### Incorrect Node Versions

To match our production environment, we specify our local environment to use a specific version of Node. If you don't already have that version, switching Node versions can be a hassle.

##### OSX

There are various CLIs that can assist with easily switching between versions of Node

- [NVM](https://github.com/nvm-sh/nvm/blob/master/README.md)
- [n](https://github.com/tj/n)

##### Windows

- [NVM for Windows](https://github.com/coreybutler/nvm-windows)
- [Scoop](https://github.com/lukesampson/scoop/wiki/Quick-Start)
```